spec/factories.rb

```diff
@@ -1,4 +1,8 @@
 FactoryBot.define do
+  factory :group do
+    name { "Sellers" }
```

Comment:
you should user Faker here as well

Reply:
Why this conversation was marked as resolved without a faker name?